### DISCUSSION TITLE
notes/%E5%89%8D%E7%AB%AF/%E5%A6%82%E4%BD%95%E5%AE%89%E8%A3%85nvm

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#709] [Announcements]
# notes/%E5%89%8D%E7%AB%AF/%E5%A6%82%E4%BD%95%E5%AE%89%E8%A3%85nvm

nvm是一个node的版本管理工具，可以简单操作node版本 (https://so.csdn.net/so/search?q=node版本&spm=1001.2101.3001.7020)的切换、安装、查看。。。等等，与npm不同的是，npm是依赖包的管理工具。 下载路径: https://github.com/coreybutler/nvm-win...

http://localhost:8080/notes/%E5%89%8D%E7%AB%AF/%E5%A6%82%E4%BD%95%E5%AE%89%E8%A3%85nvm.html

**Comment by zqtop** [comment]
哈哈哈
